ansible/roles/packages/tasks/debian.yaml

```diff
@@ -25,6 +25,7 @@
       - nfs-common
       - python3-cryptography
       - python3-pip
+      - python3-openssl
```

Comment:
why do we need this?

Reply:
fixing openssl error for ubuntu 20 builds. https://teamcity.mesosphere.io/buildConfiguration/ClosedSource_KonvoyImageBuilder_E2eUbuntu2004Azure/3608941?buildTab=log&focusLine=1703&linesState=516

This library has the python `crypto` module that has the fix. however I just checked, this didnt fixed it. Strangely it only breaks for Azure Ubuntu 20.04 build but passes for AWS Ubuntu 20.4 builds. I havent got chance to debug it again.

Reply:
upgrading pyopenssl module fixed it. so I have reset this change back.